src/Components/PresentationalComponents/Snippets/SnippetWithPopover.js

```diff
@@ -16,7 +16,7 @@ const SnippetWithPopover = props => {
         status_text: statusJustification,
         status_id: systemStatusId
     } = row.attributes;
-    const hasDefaultStatus = systemStatusId === 0 && (cveStatusId === systemStatusId) && statusJustification === null;
+    const hasDefaultStatus = systemStatusId === 0 && cveStatusId === 0 && statusJustification === null;
```

Comment:
is this change intended? cause it changes the functionality

Reply:
I agree, has default status is aimed at detecting if a user has made any changes to status. `cveStatusId === systemStatusId` was used to detect if a user has not changed status so far.

Reply:
Unless I am missing something these two expressions are identical:
`systemStatusId === 0 && (cveStatusId === systemStatusId)`
`systemStatusId === 0 && cveStatusId === 0`
because only way for interpreter to get to the second part of the expression `(cveStatusId === systemStatusId)` it has to evaluate first part to true (short-circuit evaluation) which implies `systemStatusId` is always 0 if we get into this part `(cveStatusId === systemStatusId)` which means we can substitute `systemStatusId` with zero without changing functionality.

I thought it made the expression a bit simpler, but if you think the former conveyed the meaning better I can revert it.

Reply:
@leSamo sorry!! what I was thinking? you're TOTALLY right!!! :D :D

Reply:
Well this is that kind of confusion I wanted to get rid of by this change :smile: